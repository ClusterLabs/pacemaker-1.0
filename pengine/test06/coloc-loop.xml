<?xml version="1.0"?>
<!DOCTYPE cib SYSTEM "crm-1.0.dtd">
<!-- shutdown -->
<cib num_updates="1" epoch="1" admin_epoch="0">
  <configuration>
    <crm_config><cluster_property_set id="cib-bootstrap-options"><attributes>
      <nvpair id="no_quorum_policy" name="no_quorum_policy" value="ignore"/>
      <nvpair id="symmetric_cluster" name="symmetric_cluster" value="true"/>
      <nvpair id="default_resource_failcount" name="default-resource-failure-stickiness" value="-INFINITY"/>
    </attributes></cluster_property_set></crm_config>
    <nodes>
      <node id="node1" uname="node1" type="member"/>
      <node id="node2" uname="node2" type="member"/>
      <node id="node3" uname="node3" type="member"/>
    </nodes>
    <resources>
      <primitive id="rsc1" class="heartbeat" type="apache"/>
      <primitive id="rsc2" class="heartbeat" type="apache" />
      <primitive id="rsc3" class="heartbeat" type="apache" />
    </resources>
    <constraints>
      <rsc_colocation id="colo-1-2" from="rsc1" to="rsc2" score="INFINITY"/>
      <rsc_colocation id="colo-2-3" from="rsc2" to="rsc3" score="INFINITY"/>
      <rsc_colocation id="colo-3-1" from="rsc3" to="rsc1" score="INFINITY"/>
    </constraints>
  </configuration>
  <status>
    <node_state id="node1" ha="active" uname="node1" crmd="online" join="member" expected="member" in_ccm="true">
       <transient_attributes id="node1">
         <instance_attributes id="status-node1">
           <attributes>
             <nvpair id="status-node1-fail-count-rsc4" name="fail-count-rsc4" value="1"/>
           </attributes>
         </instance_attributes>
       </transient_attributes>
       <lrm id="node1">
         <lrm_resources>
         </lrm_resources>
       </lrm>
    </node_state>
    <node_state id="node2" ha="active" uname="node2" crmd="online" join="member" expected="member" in_ccm="true">
       <transient_attributes id="node2">
         <instance_attributes id="status-node2">
           <attributes>
             <nvpair id="status-node2-fail-count-rsc3" name="fail-count-rsc3" value="1"/>
           </attributes>
         </instance_attributes>
       </transient_attributes>
    </node_state>
    <node_state id="node3" ha="active" uname="node3" crmd="online" join="member" expected="member" in_ccm="true">
       <transient_attributes id="node3">
         <instance_attributes id="status-node3">
           <attributes>
             <nvpair id="status-node3-fail-count-rsc4" name="fail-count-rsc2" value="1"/>
           </attributes>
         </instance_attributes>
       </transient_attributes>
    </node_state>
  </status>
</cib>
