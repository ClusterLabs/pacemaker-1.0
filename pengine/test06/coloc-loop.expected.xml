<?xml version="1.0" encoding="UTF-8"?>
<cib admin_epoch="0" epoch="1" num_updates="1" dc-uuid="0" have-quorum="false" remote-tls-port="0" validate-with="pacemaker-1.0">
  <configuration>
    <crm_config><cluster_property_set id="cib-bootstrap-options"><nvpair id="nvpair" name="no-quorum-policy" value="ignore"/><nvpair id="nvpair" name="symmetric-cluster" value="true"/><nvpair id="nvpair" name="default-resource-failure-stickiness" value="-INFINITY"/></cluster_property_set></crm_config>
    <nodes>
      <node id="node1" uname="node1" type="member"/>
      <node id="node2" uname="node2" type="member"/>
      <node id="node3" uname="node3" type="member"/>
    </nodes>
    <resources>
      <primitive id="rsc1" class="heartbeat" type="apache"><meta_attributes id="primitive-rsc1.meta"/></primitive>
      <primitive id="rsc2" class="heartbeat" type="apache"><meta_attributes id="primitive-rsc2.meta"/></primitive>
      <primitive id="rsc3" class="heartbeat" type="apache"><meta_attributes id="primitive-rsc3.meta"/></primitive>
    </resources>
    <constraints>
      <rsc_colocation id="colo-1-2" rsc="rsc1" with-rsc="rsc2" score="INFINITY"/>
      <rsc_colocation id="colo-2-3" rsc="rsc2" with-rsc="rsc3" score="INFINITY"/>
      <rsc_colocation id="colo-3-1" rsc="rsc3" with-rsc="rsc1" score="INFINITY"/>
    </constraints>
  </configuration>
  <status>
    <node_state id="node1" ha="active" uname="node1" crmd="online" join="member" expected="member" in_ccm="true">
       <transient_attributes id="node1">
         <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="fail-count-rsc4" value="1"/></instance_attributes>
       </transient_attributes>
       <lrm id="node1">
         <lrm_resources id="lrm_resources">
         </lrm_resources>
       </lrm>
    </node_state>
    <node_state id="node2" ha="active" uname="node2" crmd="online" join="member" expected="member" in_ccm="true">
       <transient_attributes id="node2">
         <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="fail-count-rsc3" value="1"/></instance_attributes>
       </transient_attributes>
    </node_state>
    <node_state id="node3" ha="active" uname="node3" crmd="online" join="member" expected="member" in_ccm="true">
       <transient_attributes id="node3">
         <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="fail-count-rsc2" value="1"/></instance_attributes>
       </transient_attributes>
    </node_state>
  </status>
</cib>
